index.js

```diff
@@ -2404,13 +2472,27 @@
   //. > S.reduce(S.add, 0, [1, 2, 3, 4, 5])
   //. 15
   //.
-  //. > S.reduce((xs, x) => [x].concat(xs), [], [1, 2, 3, 4, 5])
+  //. > S.reduce(xs => x => [x].concat(xs), [], [1, 2, 3, 4, 5])
   //. [5, 4, 3, 2, 1]
   //. ```
   S.reduce =
   def('reduce',
       {b: [Foldable]},
       [$.Function, a, b, a],
+      function(f_, initial, foldable) {
+        var f = function(a, b) {
+          return f_(a)(b);
+        };
+        return S.reduce_(f, initial, foldable);
```

Comment:
Let's reference this as `reduce_` (by introducing an alias) rather than as `S.reduce_`.